x-pack/plugins/maps/common/descriptor_types/sources.ts

```diff
@@ -95,16 +95,33 @@ export type XYZTMSSourceDescriptor = AbstractSourceDescriptor &
     urlTemplate: string;
   };
 
+export enum MVTFieldType {
```

Comment:
How about moving this definition to constants since that is where other enums are defined